erp/views.py

```diff
@@ -208,28 +208,24 @@ def _cleaned_search_params_as_dict(get_parameters):
 
 def _clean_address(where):
     """
-    where is a string as returned by geoloc API on frontend side. It returns city and postal_code, this is pure string
+    where is a string as returned by geoloc API on frontend side. It returns city and code_departement, this is pure string
```

Comment:
I would have use an english var name for code_departement.
Everything else is OK

Reply:
Yes I did not had any idea how to call this :thinking:

Reply:
I would have naively use `department` but according to wordreference, it can be left untranslated